l8_collections/final_project/utils.py

```diff
@@ -0,0 +1,147 @@
+import random
+import csv
+import re
+
+# creation of alphabet list
+def alph_list_creation():
+    alph_list = []
+    for i in range(ord('a'), ord('z')+1):
+        i = chr(i)
+        alph_list.append(i)
+    return alph_list
+
+# creation of names list
+def name_list_creation(alph_list):
+    name_item = []
+    for i in range(100):
+        r_num_char = random.randint(3,8)
+        for j in range(r_num_char):
+            any_char = random.choice(alph_list)
+            name_item.append(any_char)
+        name_item.append('/')
+
+    name_list = (''.join(name_item)).split('/')
+    name_list.pop()
+    return name_list
+
+# creation of phone list
+def phone_list_creation(alph_list):
+    phone_item = []
+    for i in range(100):
+        chance = random.randint(1, 50)
+        if chance == 9:
+            ran_symbol = random.choice(['[', '-', ']', ' ', random.choice(alph_list)])
+            ran_temp = '+38' + str(ran_symbol)
+            phone_item.append(str(ran_temp))
+            for k in range(8):
+                r_num = random.randint(0, 9)
+                phone_item.append(str(r_num))
+        else:
+            phone_item.append('+380')
+            for k in range(8):
+                r_num = random.randint(0,9)
+                phone_item.append(str(r_num))
+        phone_item.append('/')
+
+    phone_list = (''.join(phone_item)).split('/')
+    phone_list.pop()
+    return phone_list
+
+# creation of exp date
+def exp_list_creation():
+    exp_item = []
+    for i in range(100):
+        r_num = random.randint(1,12)
+        if r_num < 10:
+            r_num = '0'+ str(r_num)
+        r_date = random.randint(19,30)
+        exp_item.append(str(r_num))
+        exp_item.append('/')
+        exp_item.append(str(r_date))
+        exp_item.append('//')
+
+    exp_list = (''.join(exp_item)).split('//')
+    exp_list.pop()
+    return exp_list
+
+# creation of card
+def card_list_creation(alph_list):
+    card_item = []
+    for i in range(100):
+        for j in range(4):
+            chance = random.randint(1, 100)
+            if chance == 9:
+                ran_symbol = random.choice(['-', ' ', random.choice(alph_list)])
+                card_item.append(str(ran_symbol))
+                for l in range(3):
+                    r_card_num = random.randint(0, 9)
+                    card_item.append(str(r_card_num))
+                card_item.append(' ')
+            else:
+                for l in range(4):
+                    r_card_num = random.randint(0,9)
+                    card_item.append(str(r_card_num))
+                card_item.append(' ')
+        card_item.append('/')
+
+    card_list = (''.join(card_item)).split(' /')
+    card_list.pop()
+    return card_list
+
+# checking the correctness of data
+def tel_checking(phone):
+    while True:
+        checking = re.match(r'^\+[0-9]{11}$', phone)
+        if checking is None:
```

Comment:
+        if checking is None:
+            return 'Present'
+        else:
+            return  'Absent'